inc/storefront-template-functions.php

```diff
@@ -183,8 +183,8 @@ function storefront_site_branding() {
 	 */
 	function storefront_primary_navigation() {
 		?>
-		<nav id="site-navigation" class="main-navigation" role="navigation" aria-label="<?php esc_html_e( 'Primary Navigation', 'storefront' ); ?>">
-		<button class="menu-toggle" aria-controls="primary-navigation" aria-expanded="false"><span><?php echo esc_attr( apply_filters( 'storefront_menu_toggle_text', __( 'Menu', 'storefront' ) ) ); ?></span></button>
+		<div id="site-navigation" class="main-navigation" role="navigation" aria-label="<?php esc_html_e( 'Primary Navigation', 'storefront' ); ?>">
```

Comment:
> Won't that be harmful?

Was it harmfull to use `<div>` in this context before HTML5 ? :p
But we're in HTML5 now and something I'm not aware of maybe changed...
What I'm sure about is that if we're going to keep `<nav>` then we should better leave it like it was, don't add tricky hidden heading there.
Your turn for the final decision.